src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -225,7 +225,7 @@ define([
                         }
                     }),
                     error: lang.hitch(this, function(error) {
-                        logger.error(this.id + ": An error ocurred while executing nanoflow: ", error);
+                        mx.ui.error("An error ocurred while executing nanoflow");
```

Comment:
You need to specify the error message like
`mx.ui.error("An error ocurred while executing nanoflow" + error.message);` and i think it makes sense to apply this error message format to the microflow execution too